include/boost/beast/core/bind_handler.hpp

```diff
@@ -23,18 +23,18 @@ namespace beast {
 
     This function creates a new handler which, when invoked, calls
     the original handler with the list of bound arguments. Any
-    parameters passed in the invocation will be subtituted for
+    parameters passed in the invocation will be substituted for
     placeholders present in the list of bound arguments. Parameters
     which are not matched to placeholders are silently discarded.
+
     The passed handler and arguments are forwarded into the returned
-    handler, which provides the same `io_context` execution guarantees
-    as the original handler.
+    handler, which inherits the associated allocator and associated
```

Comment:
I think inherits might be misleading because it might cause the reader to think that the binder **inherits from** the associated allocator and executor. Perhaps **forwards** is a better name for this relationship?

Reply:
true, willfix